src/PHPUnit/Constraint/Page.php

```diff
@@ -22,36 +40,30 @@ public function __construct($string, $uri = '')
      *
      * @return bool
      */
-    protected function matches($other) : bool
+    protected function matches($other): bool
```

Comment:
$other is a string.

Reply:
```diff
-     * @param mixed $other Value or object to evaluate.
+     * @param string $other Value or object to evaluate.
```
the `PHPUnit/Framework/Constraint/Constraint` base class doesn't have that parameter typed, I know the only use it is given here is as a string, but that generates a lot of warnings in the IDE and static code analyzers.

Reply:
Making this change also generates errors in the tests:

```shell
1) PhpBrowserTest: See text after js comparision operator
 Test  tests/unit/Codeception/Module/PhpBrowserTest.php:testSeeTextAfterJSComparisionOperator
                                                                                                                  
  [PHPUnit\Framework\Exception] Declaration of Codeception\PHPUnit\Constraint\Page::matches(string $other): bool should be compatible with PHPUnit\Framework\Constraint\Constraint::matches($other): bool at ../../../src/PHPUnit/Constraint/Page.php:42  
```